### ISSUE TITLE
API endpoint for personal and group statistics

### ISSUE THREAD

**Issue by AlexanderFillbrunn** [hoelsch/fitness-app-backend#2]
The API needs and endpoint for statistics because they cannot be calculated locally.

**Comment by hoelsch** [comment]
Yes, I agree. Which statistics do you have in mind?

I would like to have the total weight lifted by an user.

**Comment by AlexanderFillbrunn** [comment]
Yes, total weight over time would be nice. Also nice would be trainings per week.

**Comment by AlexanderFillbrunn** [comment]
It would also be nice to have statistics for individual exercises to see the progress I make.
